frame/support/src/lib.rs

```diff
@@ -874,6 +874,7 @@ pub mod tests {
 
 	decl_storage! {
 		trait Store for Module<T: Config> as Test {
+			pub Value get(fn value): u64;
```

Comment:
One tests fails now so you have to fix the hard-coded metadata https://github.com/paritytech/substrate/blob/c19838924ed941de28ea9d6ec25a74ccf207bc65/frame/support/src/lib.rs#L1306

Reply:
Or you define it in your test like:  
```rust
#[frame_support::storage_alias]
pub type Value = StorageValue<Prefix, u32, ValueQuery>;
```

Reply:
Done, I hope it's fine now